client/my-sites/stats/activity-log/index.jsx

```diff
@@ -344,7 +345,28 @@ class ActivityLog extends Component {
 	}
 
 	render() {
-		const { isPressable, isRewindActive, siteId, siteTitle, slug } = this.props;
+		const {
+			canViewActivityLog,
+			isPressable,
+			isRewindActive,
+			siteId,
+			siteTitle,
+			slug,
+			translate,
+		} = this.props;
+
+		if ( false === canViewActivityLog ) {
+			return (
+				<Main wideLayout>
```

Comment:
I don't think this `Main` needs to have a `wideLayout` for that case - should be just fine to omit it.

Reply:
Dropped it, thanks!